app/views/representations/update.js.erb

```diff
@@ -0,0 +1,2 @@
+var element = document.getElementsByClassName("timeline-item representation-<%= @representation.id %>")[0]
+element.parentNode.removeChild(element);
```

Comment:
Is this the right behavior 100% of the time? It assumes the item should disappear from the parent view.

I'd recommend re-rendering the list here and having simple_form_for replacing the contents of the list with the newly rendered partial; otherwise we're going to be replicating all of our business logic from Rails in JavaScript.